### PR TITLE
Pankaj

### PR DESCRIPTION
**Pull request by Pankaj-Kumar110**
Added user history tab and removed unnecessary elements from UI

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="" vcs="Git" />
+  </component>
+</project>
```
Comment (Akshay-Tomar-1135) [inline .idea/vcs.xml:6]: ye aur iske upar ke changes kaise aaye?

File: package-lock.json
Comment (Akshay-Tomar-1135) [inline package-lock.json:1]: ye extra dependencies kyu? abhi tak inka use bhi start nhi hua hai is branch me. Baad me confilicts aa sakte hai. So remove them.

File: package.json
Comment (Akshay-Tomar-1135) [inline package.json:1]: why mapbox, react-google-map/api are added, if there is no use of them yet in the branch?
why react-redux is removed? It is an important dependency for redux implementation.
Don't remove existing and add new dependency unnecessarily.

File: src/_mock/_userHistory.ts
Comment (Akshay-Tomar-1135) [inline src/_mock/_userHistory.ts:1]: don't restructure these file in the _mock folder. Things will be removed and utilized further, when necessary.

File: src/_mock/enums.ts
Comment (Akshay-Tomar-1135) [inline src/_mock/enums.ts:1]: don't delete this file

File: src/_mock/index.ts [hunk 2]
```diff
@@ -1,3 +1,2 @@
 export * from './_mock';
 export * from './_data';
-export * from './enums';
```
Comment (Akshay-Tomar-1135) [inline src/_mock/index.ts:3]: do it as before

File: src/app.tsx [hunk 3]
```diff
@@ -1,11 +1,14 @@
...
+
 import { Router } from 'src/routes/sections';
+
 import { useScrollToTop } from 'src/hooks/use-scroll-to-top';
+
 import { ThemeProvider } from 'src/theme/theme-provider';
+
 import { Iconify } from 'src/components/iconify';
-import { Provider } from 'react-redux';
-import store from './store/store';
```
Comment (Akshay-Tomar-1135) [inline src/app.tsx:8]: don't remove these.
This is for redux implementation.

File: src/app.tsx [hunk 4]
```diff
@@ -33,11 +36,8 @@ export default function App() {
   );
 
   return (
-    <Provider store={store}>
-      <ThemeProvider>
-        <Router />
-        {githubButton}
-      </ThemeProvider>
-    </Provider>
```
Comment (Akshay-Tomar-1135) [inline src/app.tsx:41]: don' t remove provider.
this is to provide redux state access to the application.

File: src/components/FAQ.tsx [hunk 5]
```diff
@@ -0,0 +1,74 @@
+import { useState } from 'react';
+
+const FAQSection = () => {
+  const [activeIndex, setActiveIndex] = useState<number | null>(null);
+
+  const toggleAccordion = (index: number) => {
+    setActiveIndex(index === activeIndex ? null : index);
+  };
+
+  const faqs = [
```
Comment (Akshay-Tomar-1135) [inline src/components/FAQ.tsx:10]: you can move this 'faqs' constant to _mock/_data.ts

File: src/components/FAQ.tsx [hunk 6]
```diff
@@ -0,0 +1,74 @@
+import { useState } from 'react';
+
+const FAQSection = () => {
+  const [activeIndex, setActiveIndex] = useState<number | null>(null);
```
Comment (Akshay-Tomar-1135) [inline src/components/FAQ.tsx:4]: can you use activeIndex value as -1, instead of using null type. It will be simpler this way.

File: src/components/FAQ.tsx [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+      className="py-12 px-4 sm:px-6 md:px-8 lg:px-10 bg-gradient-to-t from-[#e0f2ff] to-[#a0c8f0] text-black"
+    >
+      <div className="flex flex-col items-center max-w-5xl mx-auto">
+        <div className="text-3xl sm:text-4xl md:text-5xl lg:text-6xl pt-6 font-bold tracking-tighter mb-8 text-center bg-gradient-to-b from-black to-[#6a0dad] text-transparent bg-clip-text">
+          Frequently Asked Questions
+        </div>
+
+        <div className="w-full space-y-4">
+          {faqs.map((faq, index) => (
+            <div
```
Comment (Akshay-Tomar-1135) [inline src/components/FAQ.tsx:41]: Weren't you going to use accordion for this?

File: src/components/Header.tsx [hunk 8]
```diff
@@ -1,53 +1,51 @@
...
+import Logo from "/assets/images/others/main-logo.png";
+import { FaBars } from "react-icons/fa";
+import { useState } from "react";
+import Box from "@mui/material/Box";
 import { useRouter } from 'src/routes/hooks';
-import { useAppDispatch } from 'src/store/reduxHooks';
-import { setUserType } from 'src/store/features/userConfig/userConfigSlice';
-import { UserTypes } from 'src/_mock';
-import UserTypeModal from './UserTypeModal';
-import Button from './Button';
```
Comment (Akshay-Tomar-1135) [inline src/components/Header.tsx:10]: revert these changes

File: src/components/Header.tsx [hunk 9]
```diff
@@ -1,53 +1,51 @@
...
+import Button from "./Button";
 
-export default function Header() {
-  const [localUserType, setLocalUserType] = useState<UserTypes | null>(null);
-  const [openModal, setOpenModal] = useState<boolean>(false);
```
Comment (Akshay-Tomar-1135) [inline src/components/Header.tsx:14]: revert this one too

File: src/components/Header.tsx
Comment (Akshay-Tomar-1135) [inline src/components/Header.tsx:1]: I guess you should revert the whole file

File: src/components/UserTypeModal.tsx
Comment (Akshay-Tomar-1135) [inline src/components/UserTypeModal.tsx:1]: revert this file removal

File: src/components/logo/logo.tsx
Comment (Akshay-Tomar-1135) [inline src/components/logo/logo.tsx:1]: are you sure, we can remove this file?

File: src/interface/UserInfoTypes.ts
Comment (Akshay-Tomar-1135) [inline src/interface/UserInfoTypes.ts:1]: revert this file

File: src/components/logo/index.ts
Comment (Akshay-Tomar-1135) [inline src/components/logo/index.ts:1]: you can remove whole logo folder.

File: src/config-global.ts [hunk 10]
```diff
@@ -10,6 +10,6 @@ export type ConfigValue = {
 // ----------------------------------------------------------------------
 
 export const CONFIG: ConfigValue = {
-  appName: 'Minimal UI',
+  appName: '',
```
Comment (Akshay-Tomar-1135) [inline src/config-global.ts:13]: You should put tha name as Suraksha, if you have removed 'Minimal UI'

File: src/layouts/auth/layout.tsx [hunk 11]
```diff
@@ -49,7 +49,6 @@ export function AuthLayout({ sx, children, header }: AuthLayoutProps) {
                 This is an info Alert.
               </Alert>
             ),
-            leftArea: <Logo />,
```
Comment (Akshay-Tomar-1135) [inline src/layouts/auth/layout.tsx:52]: have you used the Suraksha app logo, after removing this?

File: src/sections/auth/OTPVerification.tsx
Comment (Akshay-Tomar-1135) [inline src/sections/auth/OTPVerification.tsx:1]: revert this file

**Review by Akshay-Tomar-1135 — COMMENTED** [review]
take the fresh pull than apply your changes.

File: src/_mock/_userHistory.ts
Comment (Pankaj-Kumar110) [reply src/_mock/_userHistory.ts:1]: gupta bola hai karne ko

File: src/_mock/enums.ts
Comment (Pankaj-Kumar110) [reply src/_mock/enums.ts:1]: maine nhi kiya

File: src/components/FAQ.tsx (hunk 7 above)
Comment (Pankaj-Kumar110) [reply src/components/FAQ.tsx:41]: kiya hai saar

File: src/_mock/_userHistory.ts
Comment (Akshay-Tomar-1135) [reply src/_mock/_userHistory.ts:1]: koi nhi rahne de aise hi main baad me kar lunga

File: src/_mock/enums.ts
Comment (Akshay-Tomar-1135) [reply src/_mock/enums.ts:1]: yeah I understand

File: src/components/FAQ.tsx (hunk 7 above)
Comment (Akshay-Tomar-1135) [reply src/components/FAQ.tsx:41]: material UI ka accordion component. Its your choice whatever is comfortable with you
